cmd/deploy/deploy.go

```diff
@@ -274,6 +270,39 @@ func deployApp(ctx context.Context, appVersionOutput app.CreateAppVersionOutput,
 	return nil
 }
 
+type statusUpdater struct {
+	verbose               bool
+	lastStatus            *string
+	sameStatusUpdateCount int
+	task                  *output.Task
+}
+
+func (s *statusUpdater) update(status string) error {
+	if s.verbose {
+		if s.lastStatus != nil && *s.lastStatus != status {
+			s.task.UpdateLine("Deploy", "Workload is "+strings.ToLower(*s.lastStatus)+strings.Repeat(".", s.sameStatusUpdateCount)+"\n")
```

Comment:
Can be extracted into a separate function and reused below, e.g. `func formatStatus(status string, updateCount uint) string`.

Reply:
Changed in 1d22137ce6d2b61c11d3471d2caa7aa724490d10